src/popup.js

```diff
@@ -0,0 +1,18 @@
+const results = document.getElementById("results");
```

Comment:
I think this is unused and also `undefined`?

Reply:
Oops. You are right.